### ISSUE TITLE
Reject reassignment of options

### ISSUE THREAD

**Issue by mark-rushakoff** [influxdata/flux#918]
Given a task script with multiple `option task` assignments:

```
option task = {name: "mytask"}
from(bucket:"x") |> range(start: -1h) |> to(bucket: "y")
option task = {every: 1d}
```

When task code attempts to extract the option, like so:

```go
	_, scope, err := flux.Eval(script)
	if err != nil {
		return opt, err
	}

	task, ok := scope.Lookup("task")
	if !ok {
		return opt, errors.New("missing required option: 'task'")
	}
	optObject := task.Object()

	// ...
```

It only observes the latter option `every: 1d`.

It would be better if `flux.Eval` rejected the script on the grounds that `option task` is reassigned.

**Comment by jlapacik** [comment]
Options are defined to be mutable package-level variables of a single type, and so right now they can be reassigned a new value. By disallowing option reassignments in the same package block, the only way to assign a new value to an option would be from an external package as follows:

```
package a
option x = 0

package b
import "a"
option a.x = 1
```

Is the above example the only use case for options?

**Comment by russorat** [comment]
i agree that we don't really want a generic rejection of options. i'm more concerned specifically with multiple options that have special meaning for tasks.
